scripts/normalize-by-median.py

```diff
@@ -72,25 +83,32 @@ def WithDiagnostics(ifile, batch_size, fp, paired, norm):
 
             print('... in file ' + input_filename, file=sys.stderr)
 
-            if report_fp:
+            if fp:
```

Comment:
change recommended by @luizirber from the last pull:
https://github.com/dib-lab/khmer/commit/a163fa645a8938d557d0e4a586741fddf36b59cf#commitcomment-11477819